stratum/hal/lib/phal/attribute_database_mock.h

```diff
@@ -44,6 +44,8 @@ class QueryMock : public Query {
   MOCK_METHOD2(Subscribe,
                ::util::Status(std::unique_ptr<ChannelWriter<PhalDB>> subscriber,
                               absl::Duration polling_interval));
+
+  // Send back fake values to subscriptions somehow?
```

Comment:
Max, the original PR I created had code to do this in the QueryMock class, used a Subscribe function in the Mock class to call the Mock DoSubscribe function.

Reply:
I got it. gMock as the `Invoke` pattern for this.